fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -147,7 +147,8 @@ public static DataProperty analyzeDataProperty(Map<String, String> properties, D
 
         if (storageMedium == TStorageMedium.SSD && !hasCooldown) {
             // set default cooldown time
-            coolDownTimeStamp = currentTimeMs + Config.storage_cooldown_second * 1000L;
+            coolDownTimeStamp = Config.storage_cooldown_second <= 0 ? DataProperty.MAX_COOLDOWN_TIME_MS :
```

Comment:
i think it should consider overflow scenarios if already do a check

Reply:
ok,but I think what we need is a new Config class which can specify the min, max, default etc. value of one configuration. We have far more configurations which may cause bigger problems if overflow happens, and there configurations currently don't have any overflow check.